sources/web/datalab/polymer/components/datalab-toolbar/datalab-toolbar.html

```diff
@@ -83,15 +83,15 @@
 
     <app-header slot="header" fixed shadow>
       <app-toolbar>
-        <div main-title class="title">
+        <div id="title" main-title class="toolbar-title">
           <!--TODO: @yebrahim replace this with SVG version of the logo-->
-          <img src="images/datalab-logo.png" class="title-image">
+          <img src="/images/datalab-logo.png" class="title-image">
```

Comment:
Why are these absolute links now? That will cause issues if the base path is not empty.